### PR TITLE
rename pass variable to passwd

### PR DESCRIPTION
**Pull request by mrybas**
When trying to define a variable "pass" at including role uchiwa:

```
---                                                                                                                                                                                                                 
- hosts: uchiwa*                                                                                                                                                                                                    
  vars:                                                                                                                                                                                                             
    user: "admin"                                                                                                                                                                                                   
    pass: "somepassword"                                                                                                                                                                                      
  roles:                                                                                                                                                                                                            
    - role: uchiwa   
```

```
ERROR! Invalid variable name in vars specified for Play: 'pass' is not a valid variable name

The error appears to have been in '/Users/wizard/projects/sensu/playbooks/uchiwa.yml': line 4, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  vars:
      user: "admin"
          ^ here
```

### CODE REVIEW

File: tasks/repo/yum.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - name: Add the official sensu repository
   copy:
-    src='../../files/{{ sensu_yum_repo }}.repo'
```
Comment (queeno) [inline tasks/repo/yum.yml:8]: Hey, thanks very much for your PR!

Could you please put this change in a separate commit?

Otherwise it looks good to me! 👍

**Comment by mrybas** [comment]
Done

**Comment by queeno** [comment]
Thanks @mrybas 

One last thing, could you please rebase your fork on the main repo? That should get rid of the merge commit. Thanks very much!
